.ci/Makefile

```diff
@@ -18,6 +21,14 @@ clean:
 env:
 	env
 
+# Create Docker image
+build-docker: validate-branch-name
+	docker build -t $(DOCKER_REGISTRY)/$(DOCKER_IMAGE_NAME):$(BRANCH_NAME) \
+  --build-arg AGENT_DIR=$(DIST_DIR) ..
+
+push-docker: build-docker
+	../dev-utils/push-docker.sh $(DOCKER_REGISTRY) $(DOCKER_IMAGE_NAME) $(BRANCH_NAME)
```

Comment:
Forgot to 'git add dev-utils/push-docker.sh'?

Reply:
yes, forgot it, added now